pkg/common/config/config.go

```diff
@@ -293,15 +293,16 @@ func validateConfig(ctx context.Context, cfg *Config) error {
 			vcConfig.InsecureFlag = cfg.Global.InsecureFlag
 		}
 	}
-
+	clusterFlavor, err := GetClusterFlavor(ctx)
```

Comment:
I think you do not want to include this in this PR.

Reply:
yes i've updated the commits